### PR TITLE
Card class

### PR DESCRIPTION
**Pull request by mona-jena**
Fixed all current test cases. Can you please check if the cases for Game class methods have been tested for and if the way that the current tests are testing is being done correctly. Thank you

### CODE REVIEW

File: BlackJackGame/Game.cs [hunk 1]
```diff
@@ -138,19 +148,114 @@ public int CalculateScore(List<string> playersHand)
         }
         
 
-        public List<string> DealersTurn(List<string> dealersHand, IDeck deck)
+        public int DealersTurn(List<Card> dealersHand, IDeck deck, int score)
```
Comment (amandahogan) [inline BlackJackGame/Game.cs:151]: what is the int score parameter that is being passed into DealersTurn?

File: BlackJackTests/GameTests.cs [hunk 2]
```diff
@@ -210,99 +214,136 @@ public void TestIfCalculateScoreReturnsTotalSumOfPlayersHandWithFiveCards(string
...
-            var deckMock = new Mock<IDeck>();
-            deckMock.Setup((s => s.DrawCard()))
-                .Returns("Queen of Diamonds");
+                string expected = string.Format("Player Wins!{0}", Environment.NewLine);
+                Assert.Equal(expected, sw.ToString());
+            }
+        }*/
+        
+        
+        [Theory]
```
Comment (amandahogan) [inline BlackJackTests/GameTests.cs:314]: what's the difference between [Theory] and [Fact]?

File: BlackJackGame/Game.cs (hunk 1 above)
Comment (mona-jena) [reply BlackJackGame/Game.cs:151]: It is the initial score for the dealers hands, so the total score after first two cards are drawn.

File: BlackJackTests/GameTests.cs (hunk 2 above)
Comment (mona-jena) [reply BlackJackTests/GameTests.cs:314]: Theory is to do multiple test cases for the same test method but with Fact you can only do one.

**Review by amandahogan — APPROVED** [review]

**Pull request merged**